test/distributed/test_c10d_nccl.py

```diff
@@ -1114,12 +1114,6 @@ def test_send_recv(self):
             dist.recv(recv_tensor, 0)
             self.assertEqual(send_tensor, recv_tensor)
 
-        # Test with non-contiguous tensors.
-        send_tensor_view = send_tensor.t()
```

Comment:
can we keep this test and assert values are expected on rank 1?

Reply:
Good point! When we confirm that we can officially support this use case, we should add that test back.